tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/model/internal/ProjectProviderValueLabelService.java

```diff
@@ -48,7 +50,11 @@ public boolean applicable( final ServiceContext context )
 
             final ValueProperty prop = context.find( ValueProperty.class );
 
-            if( prop != null && prop.equals( NewLiferayPluginProjectOp.PROP_PROJECT_PROVIDER ) )
+            if( prop != null &&  (
+                            prop.equals( NewLiferayPluginProjectOp.PROP_PROJECT_PROVIDER ) ||
```

Comment:
This isn't right, we shouldn't have to list all possible provider types here, since every time we add one we have to remember to come back here to do this.  We need to redesign this.